content/en/blog/_posts/2021-01-28-PSP-deprecation.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: 'What deprecation of Pod Security Policies means'
```

Comment:
Comment from SIG Docs weekly meeting:
we recommend having “replace” or “replacement” in the article title, rather than “deprecate” or “deprecation”

Reply:
One thing we struggled with here is that SIG-Auth didn't have a specific suggestion for a replacement technology. (At least from what reviews we got from them.)  So "replacement" wouldn't be quite correct for what we're actually talking about here? We tried to arrange this post to explain that the deprecation process doesn't mean it's going away immediately, and that there are other options, but we don't want to recommend a specific one. Our suggestion really is to take a look at external options and choose the right one on a case-by-case basis.

Reply:
A new commit has been pushed to suggest a new title. Thank you for the feedback and please do check it out!